llama_toolchain/memory/api/api.py

```diff
@@ -23,10 +25,25 @@
 class MemoryBankDocument(BaseModel):
     document_id: str
     content: InterleavedTextMedia | URL
-    mime_type: str
+    mime_type: str | None = None
     metadata: Dict[str, Any] = Field(default_factory=dict)
 
 
+def data_url_from_file(file_path: str) -> URL:
```

Comment:
`api.py` files can never have code and should only have datatypes. can you move this to another "common" location?